packages/core/admin/admin/src/content-manager/components/DynamicZone/components/DynamicComponent.js

```diff
@@ -248,8 +248,6 @@ const IconButtonCustom = styled(IconButton)`
   }
 `;
 
-// TODO: Delete once https://github.com/strapi/design-system/pull/858
```

Comment:
I think you were supposed to add the shadow to the component it affected, @gu-stav?

Reply:
I don't know, the comment said that we should delete the TODO once that pr was merged, and it was

Reply:
The comment meant to refactor the following code block. I'd prefer if we can keep the comment.